loadbalancer/README.md

```diff
@@ -17,28 +17,47 @@ TODO
 
 ## Usage
 
-In your client, define a publisher, wrap it with a balancing strategy, and pass
-it to a retry strategy, which returns an endpoint.  Use that endpoint to make
-requests, or wrap it with other value-add middleware.
+In your client, construct a publisher for a specific remote service, and pass
+it to a load balancer. Then, request an endpoint from the load balancer
+whenever you need to make a request to that remote service.
 
 ```go
+import (
+	"github.com/go-kit/kit/loadbalancer"
+	"github.com/go-kit/kit/loadbalancer/dnssrv"
+)
+
 func main() {
-	var (
-		fooPublisher = loadbalancer.NewDNSSRVPublisher("foo.mynet.local", 5*time.Second, makeEndpoint)
-		fooBalancer  = loadbalancer.RoundRobin(fooPublisher)
-		fooEndpoint  = loadbalancer.Retry(3, time.Second, fooBalancer)
-	)
-	http.HandleFunc("/", handle(fooEndpoint))
-	log.Fatal(http.ListenAndServe(":8080", nil))
-}
+	// Construct a load balancer for foosvc, which gets foosvc instances by
+	// polling a specific DNS SRV name.
+	p := dnssrv.NewPublisher("foosvc.internal.domain", 5*time.Second, fooFactory, logger)
+	lb := loadbalancer.NewRoundRobin(p)
+
+	// Get a new endpoint from the load balancer.
+	endpoint, err := lb.Endpoint()
+	if err != nil {
+		panic(err)
```

Comment:
If your DNS name has no instances, I believe it's much better to crash your program on startup. It's much better to do that, than to successfully start up, but leave you in a state where you can't complete any requests.